client/src/views/Options.vue

```diff
@@ -16,6 +16,14 @@ div(v-click-outside="closeOptionsMenu")
             @click="resetMessage"
           )
           span.checkmark
+      .mb1 Style
+        .choices
+          label.option Show background image
```

Comment:
I think the default for this is that the background is shown, unless a user clicks on this button and wants to remove it.

Can we change this to "Remove background image" instead?